Sources/SwiftSyntax/Trivia.swift

```diff
@@ -42,6 +42,112 @@ public struct Trivia: Sendable {
     pieces.isEmpty
   }
 
+  /// The string contents of all the comment pieces with any comments tokens trimmed.
+  public var commentValue: String {
+    var comments = [String]()
```

Comment:
Could this be an array of `Substring` instead of `String` to avoid intermediate memory allocations for the comment value pieces?